### ISSUE TITLE
R014: image reconstruction error on Skyra 3T (VE11A, 20 head+neck coil)

### ISSUE THREAD

**Issue by faesposi** [CMRR-C2P/MB#128]
Dear Support,

We have installed version R014 on our Skyra-3T system (V11A), equipped with the 16+4=20 channel head+neck coil.
When preparing the sequence based on "cmrr_mbep2d_bold", we experienced the following:
If setting MB factor = 1 (no multi-band), the sequence works. However, setting MB factor to a value of 2 causes a systematic image reconstruction error.
Previously, we had installed version R013 and found that MB=1 and MB=2 were working fine (albeit with FA=78). However, when setting MB>2 (e.g. MB=4 or MB=8), due to the MB excitation pulse exceeding the maximum voltage, we had the flip angle systematically clipped down to a value < 50 degree, with a strong impact on image quality.
With R014, when setting MB>2, again we got the warning that FA was clipped down to a value < 50 degrees. However, we then got the above mentioned image reconstruction error and no data.
In summary, with R013 we were able to set at least MB=2 and work with FA=78 with acceptable quality, with R014 any MB>1 causes an image reconstruction error, regardless of the FA clipping (occurring at MB=4 or higher).

Many thanks for any hint and support.
Fabrizio Esposito (University of Salerno)

**Comment by eauerbach** [comment]
Can you collect a savelog and send it to me?

**Comment by faesposi** [comment]
Here it is. These are the error+warning events from this morning.
I tried MB=4 and got voltage warning + reconstruction error.
I tried MB=2 and got recontruction erro.

Many thanks, Fabrizio

**Comment by faesposi** [comment]
Hi. The log file can be also found at this link:
https://www.dropbox.com/s/nq041l08cb580xk/test_r014_error_warning.log?dl=0

**Comment by eauerbach** [comment]
Ok, I believe I have fixed the issue. It seems your system does not have a GPU available, and there was a bug in one of my scripts which was not handling that situation correctly. So if you would download the updated R014 installer (dated today) from the web page and reinstall it, it should now work properly.
